pyproject.toml

```diff
@@ -207,7 +210,7 @@ dev = [
   "requests-cache",
 ]
 all = [
-  "farm-haystack[docstores,audio,crawler,preprocessing,ocr,ray,dev,onnx,beir]",
+  "farm-haystack[docstores,audio,crawler,preprocessing,ocr,ray,dev,onnx,beir,openai]",
```

Comment:
I think we can remove the `openai` here.

Reply:
Really sorry for that. When I added to core dependency, I forgot to remove from the egg aggregation.